src/uri/validator/urivalidator.rs

```diff
@@ -163,9 +163,9 @@ impl UriValidator {
                     .as_ref()
                     .map_or(false, |instance| instance.contains("response"));
```

Comment:
I guess technically the spec says it should be equal to, not that it should contain "response". Shouldn't we check for equality?

Tagging @AnotherDaniel in here to sanity check me.

Reply:
You're correct. The 'contains' is a leftover from me dumbly doing a 1:1 convert from up-java, but the spec requires equality.

Reply:
I've updated it.